packages/app/src/components/AppBase/AppBase.tsx

```diff
@@ -75,8 +75,8 @@ const AppBase = () => {
     <AppProvider>
       <AlertDisplay />
       <OAuthRequestDialog />
+      <ConfigUpdater />
```

Comment:
Does this also help that the favicon isn't loaded on each page change? :)

Reply:
This helps with the favicon not loading in the login page, not sure if it helps the favicon not loading with each page change

Reply:
I verified this, the favicon was updated in the login screen, the home screen, and then any other route/nav options.